gnome-initial-setup/pages/language/gis-language-page.ui

```diff
@@ -6,33 +6,54 @@
       <object class="GtkBox" id="box">
         <property name="visible">True</property>
         <property name="orientation">vertical</property>
-        <property name="halign">center</property>
-        <property name="valign">fill</property>
-        <child>
-          <object class="GtkImage" id="logo">
-            <property name="visible" bind-source="GisLanguagePage" bind-property="small-screen" bind-flags="invert-boolean|sync-create"/>
-            <property name="can_focus">False</property>
-            <property name="margin_top">24</property>
-            <property name="pixel_size">96</property>
-            <property name="icon_name">start-here-symbolic</property>
-          </object>
-        </child>
-        <child>
-          <object class="GisWelcomeWidget" id="welcome_widget">
+	<child>
+	  <object class="GtkBox" id="language_box">
             <property name="visible">True</property>
-            <property name="margin_top">18</property>
-            <property name="margin_bottom">40</property>
-            <property name="valign">center</property>
+            <property name="orientation">vertical</property>
             <property name="halign">center</property>
-          </object>
-        </child>
-        <child>
-          <object class="CcLanguageChooser" id="language_chooser">
-            <property name="margin_bottom">18</property>
-            <property name="width_request">400</property>
+            <property name="valign">fill</property>
+            <child>
+              <object class="GtkImage" id="logo">
+		<property name="visible" bind-source="GisLanguagePage" bind-property="small-screen" bind-flags="invert-boolean|sync-create"/>
+		<property name="can_focus">False</property>
+		<property name="margin_top">24</property>
+		<property name="pixel_size">96</property>
+		<property name="icon_name">start-here-symbolic</property>
```

Comment:
Hmm, actually, I didn't see this icon appear in the right place when I ran g-i-is. Is something missing on my system?

Reply:
Here is what I see for what its worth:

![screenshot from 2017-09-02 13-52-56](https://user-images.githubusercontent.com/775309/29996766-1e394cec-9037-11e7-9a3d-2d68271afa6d.png)

Reply:
Yeah, we override that icon on the `GtkImage` here: https://github.com/endlessm/gnome-initial-setup/commit/92dd4344c60f7fe7822efbcbc512aa6569cde37f